tools/jdbc/src/main/java/org/duckdb/DuckDBNative.java

```diff
@@ -46,13 +47,15 @@ public class DuckDBNative {
             Path lib_file = Files.createTempFile("libduckdb_java", ".so");
             URL lib_res = DuckDBNative.class.getResource(lib_res_name);
             if (lib_res == null) {
-                throw new IOException(lib_res_name + " not found");
+                System.load(
+                    Paths.get("../../build/debug/tools/jdbc", lib_res_name).normalize().toAbsolutePath().toString());
```

Comment:
Should this maybe get an `#ifdef DEBUG` or similar? I suppose we don't want this code in the distributed jars? Perhaps it doesn't matter though.

Reply:
Do you think adding that makes sense or not? If not I'm fine with leaving it as it is.

Reply:
I think the likelihood of it running is pretty tiny - we can revisit it if we ever go down the path of distributing the native libraries outside the default jar